geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerConnection.java

```diff
@@ -1164,6 +1190,20 @@ public void removeUserAuth(Message message, boolean keepAlive) {
     }
   }
 
+  @VisibleForTesting
+  long putSubject(Subject subject, long existingUniqueId) {
+    long uniqueId;
+    uniqueId = clientUserAuths.putSubject(subject, existingUniqueId);
```

Comment:
Please combine the assignment on this line with the declaration on the previous one. 

Incidentally, `uniqueId` can be `final`.

Reply:
done